### ISSUE TITLE
trinity compute nodes: cannot ssh into

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#261]
**Software:** master_installer 1030300
**Component:** trinity compute nodes
**Hardware:** from "a" environment: nodes001,002
**Test case:**
**Bug description:**
After netboot node with the trinity image ( nodeset node001 osimage="centos7.0_x86_64-netboot-trinity"
The nodes are reported in sshd mode (nodestat node001)
But ssh node001 gives:

```
root@controller ~]# ssh node001
Last login: Fri May 22 13:35:25 2015 from controller.cluster
/bin/bash: Permission denied
Connection to node001 closed.

```

**Steps to reproduce:**
**Expected output:**
**Actual output:**

**Comment by themisa** [comment]
selinux issue

**Comment by hansthen** [comment]
I am not sure why this issue comes up now. However it can be solved (as a workaround) by adding selinux=0 to the kernel bootparams (column addkcmdline in the bootparams table).

**Comment by hansthen** [comment]
In dtap branch I was not able to reconstruct this error. Can you try again?

**Comment by themisa** [comment]
Issue present also in *r3 
0d12eb5

**Comment by hansthen** [comment]
Fixed in d02b360c2618350feb3ee6725cef4db101062ef7. Use --with-tables to update, and all compute nodes will boot with selinux disabled. Assigning to Themis for testing.

**Comment by themisa** [comment]
depends on #286

**Comment by themisa** [comment]
fixed in dtap 7c7b15f
